mysql-test/suite/mariabackup/data_directory.test

```diff
@@ -1,8 +1,9 @@
---source include/have_symlink.inc
 let $table_data_dir=$MYSQLTEST_VARDIR/ddir;
 mkdir $table_data_dir;
+--disable_warnings
 --replace_result $table_data_dir table_data_dir
 EVAL CREATE TABLE t(a INT) ENGINE=InnoDB DATA DIRECTORY='$table_data_dir';
+--enable_warnings
```

Comment:
This test is for the "DATA DIRECTORY" option which requires symlinks to actually work. Bypassing in this way just fakes a test pass rather than fixing the actual problem (see my previous comments in this PR).